src/users/registration.md

```diff
@@ -62,11 +62,11 @@ Deis Workflow supports three registration modes:
 | admin\_only       | Only existing admins may register new users     |
 
 To modify the registration mode for Workflow you may add or modify the `REGISTRATION_MODE` environment variable for the
-controller component. If Deis Workflow is already running, use `kubectl --namespace=deis edit deployment deis-controller`:
+controller component. If Deis Workflow is already running, use:
 
-Find the `REGISTRATION_MODE` environment variable in the template file or add the appropriate section:
+`kubectl patch deployments deis-controller -p '{"spec":{"template":{"spec":{"containers":[{"name":"deis-controller","env":[{"name":"REGISTRATION_MODE","value":"disabled"}]}]}}}}'`
 
-![](../images/controller-registration-mode.png)
```

Comment:
The file?

![](https://4.bp.blogspot.com/-ubmz4Nyl8M0/Vv4I8t0CP4I/AAAAAAAAPSI/LUBSV9IepmMrCR3io7gA9ui0YFdJGiPYQ/s1600/the-matrix-neo-mouth-600x300.jpg)